website/www/site/content/en/documentation/dsls/dataframes/differences-from-pandas.md

```diff
@@ -63,7 +63,7 @@ Examples:
 Beam DataFrame operations are deferred, but the schemas of the resulting DataFrames are not, meaning that result columns must be computable without access to the data. Some DataFrame operations can’t support this usage, so they can’t be implemented. These operations raise a [WontImplementError](https://beam.apache.org/releases/pydoc/{{< param release_latest >}}/apache_beam.dataframe.frame_base.html#apache_beam.dataframe.frame_base.WontImplementError).
 
 <!-- TODO(BEAM-12169): Document the use of categorical columns as a workaround -->
-Currently there’s no workaround for this issue. But in the future, Beam Dataframe may support non-deferred column operations on categorical columns. This work is being tracked in [BEAM-12169](https://issues.apache.org/jira/browse/BEAM-12169).
+Currently there’s no workaround for this issue. But in the future, Beam Dataframe may support non-deferred column operations on categorical columns. This work is being tracked in [BEAM-12169](https://github.com/apache/beam/issues/20958).
```

Comment:
here the link text shows the JIRA issue but theaddress links to the github issue. I guess this is fine: `[BEAM-12169](https://github.com/apache/beam/issues/20958)`

Reply:
Yeah, I'm not too worried about fixing these types of instances - it doesn't make much of a difference IMO